Servers/UI/OJS.Servers.Ui/ClientApp/src/components/guidelines/buttons/ShowMoreButton.tsx

```diff
@@ -0,0 +1,45 @@
+import React, {useCallback, useState} from 'react';
+import {LinkButton} from "./Button";
+import {MdExpandLess, MdExpandMore} from "react-icons/md";
+
+import styles from './ShowMoreButton.module.scss'
+interface IShowMoreButtonProps {
+    onClick: () => void;
+}
+
+const ShowMoreButton = ({
```

Comment:
I would rename this to `StateButton`, or something like this, and move `Show less` and `Show more` into props. This will make it more reusable.